### PR TITLE
Add note that IDE0050 is now a refactoring

### PR DESCRIPTION
**Pull request by Youssef1313**
https://github.com/dotnet/roslyn/commit/584878b3903b8222f29e332a4a3c66b5fc798730

### CODE REVIEW

**Comment by Youssef1313** [comment]
FYI @CyrusNajmabadi

**Comment by gewarren** [comment]
@Youssef1313 How is the new refactoring different to this one that's been around for a few years? https://docs.microsoft.com/en-us/visualstudio/ide/reference/convert-anonymous-type-to-tuple?view=vs-2022

**Comment by Youssef1313** [comment]
The screenshot in that article actually shows IDE0050, so that is the analyzer/codefix that was documented here.

With it now being refactoring, you cannot enforce it on build or change a severity.

**Comment by gewarren** [comment]
> The screenshot in that article actually shows IDE0050, so that is the analyzer/codefix that was documented here.
> 
> With it now being refactoring, you cannot enforce it on build or change a severity.

Okay, so I guess that documentation doesn't need to be updated then.

**Comment by Youssef1313** [comment]
There is no IDE0050 anymore.

I even see @CyrusNajmabadi suggested deleting the documentation in his PR, which we may do if there is a good redirection.

**Comment by Youssef1313** [comment]
@gewarren Actually the link you provided looks like a good redirection. Makes sense to retire the article completely?

File: docs/fundamentals/code-analysis/style-rules/ide0050.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ dev_langs:
 ---
 # Convert anonymous type to tuple (IDE0050)
 
+> [!IMPORTANT]
+> This analyzer was changed to a refactoring in Visual Studio 2022 17.0
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/style-rules/ide0050.md:19]:
```suggestion
> This style rule was removed and converted to a Visual Studio refactoring in Visual Studio 2022. For information about the refactoring, see [Convert anonymous type to tuple](/visualstudio/ide/reference/convert-anonymous-type-to-tuple).
```

**Comment by gewarren** [comment]
> @gewarren Actually the link you provided looks like a good redirection. Makes sense to retire the article completely?

I don't think we should delete the article, because people may be using a NuGet analyzer package or the .NET 5 set of analyzers.

**Review by gewarren — APPROVED** [review]
Thanks for catching this!

**Pull request merged**